### PR TITLE
Added FUND and Unification

### PR DESCRIPTION
**Pull request by andreas-barthel**

### CODE REVIEW

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -2064,28 +2065,57 @@ const chainInfos = (
...
+        coinType: 5555,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("und"),
+      currencies: [
+        {
+          coinDenom: "FUND",
+          coinMinimalDenom: "nund",
+          coinDecimals: 9,
+          coinGeckoId: "unification",
+          coinImageUrl: "/tokens/fund.png",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:2105]: Please upload this image to the tokens folder

File: packages/web/config/chain-infos.ts [hunk 2]
```diff
@@ -2064,28 +2065,57 @@ const chainInfos = (
...
+        },
+      ],
+      gasPriceStep: {
+        low: 100,
+        average: 200,
+        high: 300,
       },
+      features: ["stargate", "ibc-transfer", "no-legacy-stdTx", "ibc-go"],
+      explorerUrlToTx:
+        "https://explorer.unification.chainmasters.ninja/unification/tx/${txHash}",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:2117]: please remove the '$' dollar sign

File: packages/web/config/ibc-assets.ts [hunk 3]
```diff
@@ -945,12 +945,19 @@ export const IBCAssetInfos: (IBCAsset & {
...
+    coinMinimalDenom:
+      "factory:kujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7:uusk",
     depositUrlOverride:
       "https://blue.kujira.app/ibc?destination=osmosis-1&source=kaiyo-1&denom=factory%2Fkujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7%2Fuusk",
     withdrawUrlOverride:
       "https://blue.kujira.app/ibc?destination=kaiyo-1&source=osmosis-1&denom=ibc%2F44492EAB24B72E3FB59B9FA619A22337FB74F95D8808FE6BC78CC0E6C18DC2EC",
   },
+  {
+    counterpartyChainId: "FUND-MainNet-2",
+    sourceChannelId: "channel-382",
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:957]: can we please register this IBC connection to the Cosmos Chain Registry? (https://github.com/cosmos/chain-registry/tree/master/_IBC)

File: packages/web/config/chain-infos.ts (hunk 1 above)
Comment (andreas-barthel) [reply packages/web/config/chain-infos.ts:2105]: Done

File: packages/web/config/chain-infos.ts (hunk 2 above)
Comment (andreas-barthel) [reply packages/web/config/chain-infos.ts]: Done

File: packages/web/config/ibc-assets.ts (hunk 3 above)
Comment (andreas-barthel) [reply packages/web/config/ibc-assets.ts:957]: I made a pull request:
https://github.com/cosmos/chain-registry/pull/914

**Review by JeremyParish69 — APPROVED** [review]
LGTM

**Pull request merged**